suite-native/module-accounts-import/src/components/AccountImportSummary.tsx

```diff
@@ -27,5 +27,5 @@ export const AccountImportSummary = ({ networkSymbol, accountInfo }: AccountImpo
     if (account) return <AccountAlreadyImported account={account} />;
     if (isAccountImportSupported)
         return <AccountImportSummaryForm networkSymbol={networkSymbol} accountInfo={accountInfo} />;
-    return null;
+    return null; // TODO display an error message
```

Comment:
Can you please add some simple error message here? We should have component for that.

Reply:
fixed b27328188

Reply:
fixed in 6c077a53a